arex-storage-web-api/src/main/java/com/arextest/storage/web/controller/AgentRecordingController.java

```diff
@@ -204,4 +281,24 @@ Mocker queryTest(@RequestBody AREXMocker body, @RequestHeader(name = AREX_MOCK_S
     return null;
   }
 
+  private boolean isInvalidMocker(String recordId) {
+    byte[] key = (INVALID_MOCKER_REDIS_KEY + recordId).getBytes(StandardCharsets.UTF_8);
+    return redisCacheProvider.get(key) != null;
+  }
+
+  private void setInvalidMockerRedis(String recordId) {
+    byte[] key = (INVALID_MOCKER_REDIS_KEY + recordId).getBytes(StandardCharsets.UTF_8);
+    byte[] value = recordId.getBytes(StandardCharsets.UTF_8);
+    redisCacheProvider.put(key, FIVE_MINUTES_EXPIRE, value);
```

Comment:
Detecting the existence of a key is enough to determine that it is invalid, and you don't need to care about the value of the value.  Therefore, this value should occupy a small enough space in Redis.

Reply:
Using new byte[0] instead of value represents an empty array, occupying only 16 bytes of memory.